parser-typechecker/src/Unison/Codebase/FileCodebase.hs

```diff
@@ -451,7 +455,7 @@ syncToDirectory fmtV fmtA codebase localPath branch = do
   b <- (liftIO . exists) localPath
   if b then do
     let code = codebase1 fmtV fmtA localPath
-    remoteRoot <- Codebase.getRootBranch code
+    remoteRoot <- fromMaybe Branch.empty . rightMay <$> Codebase.getRootBranch code
```

Comment:
Instead of `BranchLoadMode` with a failing option, we convert failure to empty at the call site.